api/app/src/main/kotlin/packit/controllers/PacketGroupController.kt

```diff
@@ -62,17 +64,22 @@ class PacketGroupController(
         return ResponseEntity.ok(packetGroupService.getPacketGroupSummaries(payload, filter))
     }
 
+    @PreAuthorize("@authz.canManageAnyPacket(#root)")
+    @GetMapping("packetGroups/user-role/read-permission")
```

Comment:
This is still not great and is prone to mess things up if a packetGroup is called `user-role`.

How about `packetGroups/_/read-permission`, where the underscore denotes that the endpoint is for the whole collection? I think it is reasonable to assume that no packet group will ever have that name.

(I still don't think this endpoint should even exist, but whatever)